scripts/clone_wiki_pages.py

```diff
@@ -0,0 +1,62 @@
+"""
+Create copies of wiki pages for existing forks and registrations instead of
+using the same NodeWikiPage objects as the original node.
+"""
+import logging
+import sys
+
+from framework.transactions.context import TokuTransaction
+
+from website.addons.wiki.model import NodeWikiPage
+from website.models import Node
+from website.app import init_app
+from scripts import utils as script_utils
+
+logger = logging.getLogger(__name__)
+
+
+def main():
+    nodes = Node.find()
+    clone_wiki_pages(nodes)
+
+
+def clone_wiki_pages(nodes):
+    for i, node in enumerate(nodes):
+        if node.wiki_pages_versions:
+            cloned_wiki_pages = {}
+            for key in node.wiki_pages_versions:
+                cloned_wiki_pages[key] = []
+                for wiki_id in node.wiki_pages_versions[key]:
+                    node_wiki = NodeWikiPage.load(wiki_id)
+                    if node_wiki.node._id != node._id:
+                        clone = node_wiki.clone()
```

Comment:
As mentioned on FD: can you refactor this into a method of NodeWikiPage to DRY up the code?